src/configurator/playground-configurator.ts

```diff
@@ -561,6 +572,21 @@ ${props.join('\n')}
     `;
   }
 
+  private inputKnob(knob: KnobsOfType<'input'>) {
```

Comment:
Nit: Private field names should have a _ prefix. But a big deal because this is the configurator.

Reply:
changed all private props and methods in file to have `_` prefix